tests/store/test_sqlalchemy_store.py

```diff
@@ -293,15 +294,16 @@ def test_run_info(self):
             else:
                 self.assertEqual(v, v2)
 
-    def _get_run_configs(self, name='test', experiment_id=None, tags=(), parent_run_id=None):
+    def _get_run_configs(self, name='test', experiment_id=None,
+                         tags=(), parent_run_id=None, start_time=int(time.time())):
```

Comment:
Unfortunately in Python setting `start_time=int(time.time())` will cause the default value to be initialized once & reused across function calls, instead of computed with each function call, so I'd undo this change.

Reply:
Scary stuff... 

Lemme default this to `None` and call `time.time()` inside the function, if not supplied by caller.